### DISCUSSION TITLE
HW 02, part 4, question 3

### DISCUSSION THREAD

**Discussion by feliciones** [cooperrc/computational-mechanics#53] [Q&A]
I am attempting to write the function for the problem, however, I continuously get an error message about the syntax. Does anyone know what I am doing wrong?

![image](https://user-images.githubusercontent.com/106564968/174497248-aecb3335-3aa9-4c43-a789-2fbc16821f40.png)

**Comment by deena101** [comment]
Did you define L and E?

**Comment by FishMacaron** [comment]
When calling the montecarlo function, don't put N=100, I think that will work. You don't need to put anything for N, it's just there to be a given in the function. Try montecarlo_buckle(200e9,r_mean,r_std,5)

**Comment by cooperrc** [comment]
There's a missing paranthesis on the line above the error
